Adyen/UI/Form/FormViewController.swift

```diff
@@ -130,19 +154,22 @@ open class FormViewController: UIViewController, Localizable, KeyboardObserver {
     }
     
     // MARK: - View
-
-    override open func loadView() {
-        view = formView
-        view.backgroundColor = style.backgroundColor
-    }
     
     /// :nodoc:
     override open func viewDidLoad() {
         super.viewDidLoad()
+        addFormView()
         
         delegate?.viewDidLoad(viewController: self)
         itemManager.itemViews.forEach(formView.appendItemView(_:))
     }
+
+    private func addFormView() {
```

Comment:
Why did you reverted all my changes in this file? 😅
You could have not approve my commit...

Reply:
I didn't revert everything, just the form view being loaded in the loadView(), because it was causing a weird space at the top of the card component when used outside dropIn.

I approved your PR because I liked it :-), but then found this bug.